sample-service/pom.xml

```diff
@@ -18,8 +18,9 @@
     <dependencies>
 
         <dependency>
-            <groupId>com.microsoft.azure</groupId>
-            <artifactId>azure-keyvault-secrets-spring-boot-starter</artifactId>
+            <groupId>com.azure.spring</groupId>
+            <artifactId>azure-spring-boot-starter-keyvault-secrets</artifactId>
```

Comment:
is this right? whats the difference between the two?

Reply:
yes, this is a newer one.